src/applications/disability-benefits/all-claims/pages/reservesNationalGuardService.jsx

```diff
@@ -0,0 +1,163 @@
+import { get } from 'lodash';
+
+import fullSchema from '../config/schema';
+
+import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
+import {
+  ReservesGuardDescription,
+} from '../utils';
+
+import  {
```

Comment:
There's an address in here, which maybe should be its own definition, but I figured we could leave it alone until we get an answer from EVSS on if they'll add unitAddress to their API.